common/txmgr/types/chain.go

```diff
@@ -0,0 +1,11 @@
+package types
+
+import "fmt"
+
+// SEQUENCE represents constraint on a chain's sequence object.
+// It should be convertible to a string
+type SEQUENCE fmt.Stringer
```

Comment:
Can we keep our exceptional naming limited to type parameters?
```suggestion
// Sequence represents constraint on a chain's sequence object.
// It should be convertible to a string
type Sequence fmt.Stringer
```

Reply:
These actually aren't constraints as defined either. We use them to constrain type parameters, but they are just interface types that can be used like any other.

Reply:
Done